modules/cluster-cloud-controller-manager-operator.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+[id="cluster-cloud-controller-manager-operator_{context}"]
+= Cluster Cloud Controller Manager Operator
+
+[discrete]
+== Purpose
+
+The Cluster Cloud Controller Manager Operator manages and updates the cloud controller managers deployed on top of {product-title}. The Operator is based on the Kubebuilder framework and `controller-runtime` libraries. It is installed via the Cluster Version Operator (CVO).
+
+It contains the following components:
+
+* Operator
+* Bootstrap manifest renderer
+* Installer based on static pods
```

Comment:
I think we can drop these, we don't actually do either of these anymore, plans changed a little

Reply:
+1 enhancement part for bit more context: https://github.com/openshift/enhancements/blob/master/enhancements/machine-api/out-of-tree-provider-support.md#bootstrap-changes